core/src/View/Kokkos_ViewMapping.hpp

```diff
@@ -3353,6 +3353,9 @@ KOKKOS_FUNCTION bool within_range(Map const& map,
   return (((std::size_t)indices < map.extent(Enumerate)) && ...);
 }
 
+// Disabled when using MDSpan because the MDSpan implementation has its own
+// version
+#ifndef KOKKOS_ENABLE_IMPL_MDSPAN
 template <class... Indices>
 KOKKOS_FUNCTION constexpr char* append_formatted_multidimensional_index(
```

Comment:
This function looks the exact same as in `core/src/View/Kokkos_ViewAccessPreconditionsCheck.hpp`, could we just have this in one place?

Reply:
There is an include issue here. The PreconditionCheck stuff is only included if MDSPAN is enabled.

Reply:
This file will go away eventually anyway so I am not concerned about code duplications.

Reply:
And I want to not start mixing the new implementation into the old one.